x/likenft/abci.go

```diff
@@ -0,0 +1,38 @@
+package likenft
+
+import (
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/likecoin/likechain/x/likenft/keeper"
+	"github.com/likecoin/likechain/x/likenft/types"
+)
+
+// EndBlocker called every block, process class reveal queue.
+func EndBlocker(ctx sdk.Context, keeper keeper.Keeper) {
+	defer telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), telemetry.MetricKeyEndBlocker)
+
+	logger := keeper.Logger(ctx)
+
+	// Reveal class when its reveal time is reached.
+	keeper.IterateClassRevealQueue(ctx, func(classId string) bool {
+		// Note: Create public functions from keeper files and access them here instead of mutating directly
+
+		// TODO: Mint unclaimed tokens supply to class owner #206
+
+		// TODO: Reveal nft class #206
+
+		// TODO: Remove class from queue #206
+
+		ctx.EventManager().EmitTypedEvent(&types.EventRevealClass{
```

Comment:
this should be part of business logic, not queue level, but let's leave it here for now

Reply:
add a todo for refactoring maybe?